cfme-performance/fixtures/change_forgery_protection.py

```diff
@@ -0,0 +1,36 @@
+from utils.log import logger
+import pytest
+from utils.ssh import SSHClient
+import time
+
+
+@pytest.yield_fixture(scope='session')
+def change_forgery_protection():
+    starttime = time.time()
+    ssh_client = SSHClient()
+    logger.info('Turning off "allow_forgery_protection"')
+
+    sed_command = ("sed -i \'s/allow_forgery_protection = true/allow_forgery_protection = false/\' "
+               "/var/www/miq/vmdb/config/environments/production.rb")
+    system_command = "service evmserverd restart"
+    ssh_client.run_command(sed_command)
+    ssh_client.run_command(system_command)
```

Comment:
Lets follow the same pattern here - https://github.com/akrzos/cfme-performance/blob/master/cfme-performance/utils/appliance.py#L64